applications/xbi-format-psp-t3rn.md

```diff
@@ -0,0 +1,227 @@
+# W3F Grant Proposal: PSP of XBI - an XCM-based high-level standard and interface (ABI) for smart contracts
+> This document will be part of the terms and conditions of your agreement and therefore needs to contain all the required information about the project. Don't remove any of the mandatory parts presented in bold letters or as headlines! Lines starting with a `>` (such as this one) can be removed.
+>
+> See the [Grants Program Process](https://github.com/w3f/Grants-Program/#pencil-process) on how to submit a proposal.
+
+- **Project Name:** XBI - xcm-based high-level standard and interface (ABI) for smart contracts
+- **Team Name:** t3rn
+- **Payment Address:** 0x343f822207f65fba7cc5325fd76d879528e706f4
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 2
+
+> ⚠️ *The combination of your GitHub account submitting the application and the payment address above will be your unique identifier during the program. Please keep them safe.*
+
+## Project Overview :page_facing_up:
+
+### Overview
+Propose a new Polkadot Standard Proposal for an XCM-based high-level interface for cross-chain smart contract execution - XBI.
+
+#### Brief Description
+XBI will be an XCM-based Binary Interface that extends the XCM-protocol to enable smart contracts calls, while receiving execution results back to the source Parachain. The same interface will be used to connect Smart Contract VMs installed on other Parachains, as well as to communicate with remote-to-Polkadot blockchains using the XCM protocol, which will be compatible with the bridges of the most active blockchain ecosystems today (i.e. Ethereum, Solana, Avalanche, Cosmos).
+
+To further adoption of XBI within the Polkadot ecosystem we specify XBI as a PSP - Polkadot Standard Proposal.
+
+#### Rationale
+XBI aspires to be highly practical - therefore recognizes a difference between the WASM and EVM, most popular smart contract byte code in Polkadot ecosystem as of today. As a high-level format XBI sets the  standard for interfaces implementing interaction between EVM and WASM based contracts.
```

Comment:
First and foremost, XBI focuses on usability, as such it will recognise the difference between WASM and EVM, the most popular smart contract byte code in the Polkadot ecosystem, as of today. As a high-level format XBI sets the  standard for interfaces implementing interactions between EVM and WASM based contracts.